File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/carpenterswall/domain/CarpentersWallRightObject.java
package com.zwstudio.logicpuzzlesandroid.puzzles.carpenterswall.domain;

import com.zwstudio.logicpuzzlesandroid.home.domain.HintState;

public class CarpentersWallRightObject extends CarpentersWallObject {
    public HintState state = HintState.Normal;
    public boolean isHint() {return true;}
    public String objAsString() {
        return "right";
    }
}

File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/carpenterswall/domain/CarpentersWallRightObject.kt
package com.zwstudio.logicpuzzlesandroid.puzzles.carpenterswall.domain

import com.zwstudio.logicpuzzlesandroid.home.domain.HintState

class CarpentersWallRightObject : CarpentersWallObject() {
    var state = HintState.Normal
    override val isHint: Boolean
        get() = true

    override fun objAsString(): String {
        return "right"
    }
}